_health-care/_js/components/personal-information/NameAndGeneralInfoSection.jsx

```diff
@@ -26,8 +26,7 @@ class NameAndGeneralInfoSection extends React.Component {
                   onUserInput={(update) => {this.props.onStateChange('mothersMaidenName', update);}}/>
               <SocialSecurityNumber ssn={this.props.data.socialSecurityNumber}
                   onValueChange={(update) => {this.props.onStateChange('socialSecurityNumber', update);}}/>
-              <Gender gender={this.props.data.gender}
-                  onUserInput={(update) => {this.props.onStateChange('gender', update);}}/>
+              <Gender value={this.props.data.gender}/>
```

Comment:
Oh... thanks for the clarification. I was hoping our components could appear as unadorned as possible, but it seems like this change is necessary. Will change.